code/util.py

```diff
@@ -146,19 +230,78 @@ def addMapping(item):
             newId = item['tvdbId']
         if 'tmdb_or_tvdb_Id' not in item and 'tmdbId' not in item and 'tvdbId' not in item:
             pr("Error: " + item['title'] + " has no tmdbId or tvdbId")
-            return
+            return False
     if 'season' not in item:
         item['season'] = 1
         # add mapping to mapping.csv
         pr("Adding mapping: " + item['title'] + " " + str(item['anilistId']) +
             " " + str(newId) + " " + str(item['season']))
         # if not the first line in mapping.csv, add a new line
-        if os.stat(configPath + 'mapping.csv').st_size != 0:
-            with open(configPath + 'mapping.csv', 'a') as f:
+        if os.stat(mappingFile).st_size != 0:
+            with open(mappingFile, 'a') as f:
                 f.write("\r")
-        with open(configPath + 'mapping.csv', 'a') as f:
+        with open(mappingFile, 'a') as f:
             f.write(item['title'] + ";" + str(item['anilistId']) +
                     ";" + str(newId) + ";" + str(item['season']))
+            return True
+
+
+def animeMatch(result, show):
+    print(str(result))
```

Comment:
This shouldn't print to output. If we need it, let's only dump it when logging

Reply:
Oops. That was from debugging and testing.